### PR TITLE
[Xcode12][PhotosUI] Update for beta 1

### PR DESCRIPTION
**Pull request by whitneyschmidt**

### CODE REVIEW

File: src/photosui.cs [hunk 1]
```diff
@@ -480,4 +481,92 @@ interface PHProjectTypeDescriptionInvalidator
...
+		[Export ("videosFilter")]
+		PHPickerFilter VideosFilter { get; }
+
+		[Static]
+		[Export ("livePhotosFilter")]
+		PHPickerFilter LivePhotosFilter { get; }
+
+		[Static]
+		[Export ("anyFilterMatchingSubfilters:")]
+		PHPickerFilter AnyFilterMatchingSubfilters (PHPickerFilter[] subfilters);
```
Comment (rolfbjarne) [inline src/photosui.cs:549]: This needs a verb to match our naming guidelines:

```suggestion
		PHPickerFilter GetAnyFilterMatchingSubfilters (PHPickerFilter[] subfilters);
```

File: src/photosui.cs [hunk 2]
```diff
@@ -480,4 +481,92 @@ interface PHProjectTypeDescriptionInvalidator
...
+		PHPickerFilter Filter { get; set; }
+
+		[Export ("initWithPhotoLibrary:")]
+		IntPtr Constructor (PHPhotoLibrary photoLibrary);
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (14,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface PHPickerFilter : NSCopying
```
Comment (spouliot) [inline src/photosui.cs:533]: `__attribute__((objc_subclassing_restricted))`
My guess is that we should `[Seal]` the type

Comment (spouliot) [reply src/photosui.cs:533]: makes even more sense since it's all static members

File: src/photosui.cs [hunk 3]
```diff
@@ -480,4 +481,92 @@ interface PHProjectTypeDescriptionInvalidator
...
+		PHPickerFilter LivePhotosFilter { get; }
+
+		[Static]
+		[Export ("anyFilterMatchingSubfilters:")]
+		PHPickerFilter AnyFilterMatchingSubfilters (PHPickerFilter[] subfilters);
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (14,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
```
Comment (spouliot) [inline src/photosui.cs:554]: `__attribute__((objc_subclassing_restricted))`
Let's `[Seal]` this

Comment (spouliot) [reply src/photosui.cs:554]: filed with sharpie (for future bindings) https://github.com/xamarin/ObjectiveSharpie/issues/138

Comment (whitneyschmidt) [reply src/photosui.cs:554]:
```build/ios/native/PhotosUI/PHPickerConfiguration.g.cs(85,22): warning CS0628: 'PHPickerConfiguration.PHPickerConfiguration(IntPtr)': new protected member declared in sealed class
build/ios/native/PhotosUI/PHPickerConfiguration.g.cs(109,27): error CS0549: 'PHPickerConfiguration.Copy(NSZone?)' is a new virtual member in sealed class 'PHPickerConfiguration'
build/ios/native/PhotosUI/PHPickerFilter.g.cs(71,22): warning CS0628: 'PHPickerFilter.PHPickerFilter(IntPtr)': new protected member declared in sealed class
build/ios/native/PhotosUI/PHPickerFilter.g.cs(79,27): error CS0549: 'PHPickerFilter.Copy(NSZone?)' is a new virtual member in sealed class 'PHPickerFilter'
build/ios/native/PhotosUI/PHPickerResult.g.cs(71,22): warning CS0628: 'PHPickerResult.PHPickerResult(IntPtr)': new protected member declared in sealed class
build/ios/native/PhotosUI/PHPickerViewController.g.cs(86,22): warning CS0628: 'PHPickerViewController.PHPickerViewController(IntPtr)': new protected member declared in sealed class
```

Comment (whitneyschmidt) [reply src/photosui.cs:554]: Replacing with `[Advice ("This type should not be subclassed.")]` for now :)

File: src/photosui.cs [hunk 4]
```diff
@@ -529,7 +531,7 @@ interface PHPickerConfiguration : NSCopying
 
 	[NoWatch, NoTV, NoMac, iOS (14,0)]
 	[BaseType (typeof (NSObject))]
-	[DisableDefaultCtor]
+	[Advice ("This type should not be subclassed.")]	[DisableDefaultCtor]
```
Comment (spouliot) [inline src/photosui.cs:534]: minor: extra backspace ? `[DisableDefaultCtor]` should be on it's own line

**Pull request merged**